app/services/application_purger.rb

```diff
@@ -0,0 +1,29 @@
+class ApplicationPurger
```

Comment:
Just wondering about the name - tiny thing but is it purging the application itself or just its evidence / documents. Perhasp `ApplicationEvidencePurger` might be clearer.

If you think this might be used to purge other things though I think makes sense as is.

Anywany, just a thought I don't have a massively strong opinion on this.

Reply:
Is purging both. It is the replacement for previous `current_crime_application.destroy`

It encapsulates any cleanup needed. At the moment that is evidence (plus the application itself) but there might be other cleanup required in the future thus it is not only about evidence.